### ISSUE TITLE
Some exercises need a rework

### ISSUE THREAD

**Issue by TheGP** [rust-lang/rustlings#2083]
What kind of exercise is there for Rc? There is no exercise, just some copy-paste stuff. 
The same for Arc. And by the way, I didn't read anything about Arc / Cow in the book while doing smart pointers.
For lifetimes it was too simple as well.

**Comment by TheGP** [comment]
I found the Arc, it is later in the book https://rust-book.cs.brown.edu/ch16-03-shared-state.html#shared-state-concurrency

**Comment by mo8it** [comment]
I don't find this feedback constructive. I will rename the issue to focus on one point.

**Comment by TheGP** [comment]
As there are no exercises there can't be any constructive feedback either.

**Comment by leroivi** [comment]
I agree that the Rc exercice is not really worth. As the surrounded code treats other values exactly the same way, the pattern is way too noticeable and the learner (me) can copy/paste without understanding anything.
I think the exercice shouldn't expose the usage example on others value.
Maybe we can come up with an exercice designed around a device that needs to be unique (like Audio system).

@TheGP about LifeTime : the Readme indicates that there is a lifetimekata project that propose exercises that go deeper. I think it's good enough and allow this project to put the effort on other parts.
